Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,109 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = [
+    %w[brew core cask],
+    OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") },
+    OFFICIAL_CASK_TAPS.reject { |t| t == "cask" },
+  ].flatten.freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner "`contributions` <email|name>"
+      description <<~EOS
+        Contributions to Homebrew repos for a user.
+
+        The first argument is a name (e.g. "BrewTestBot") or an email address (e.g. "brewtestbot@brew.sh").
+      EOS
+
+      comma_array "--repositories",
+                  description: "Specify a comma-separated (no spaces) list of repositories to search. " \
+                               "Supported repositories: #{SUPPORTED_REPOS.join(", ")}. " \
```

Comment:
Again, not really blocking but we could use `#to_sentence` here. Also, it might be nice to get backticks around the options so they stand out (unless you think that looks too chaotic or isn't worth the complexity)

Reply:
```suggestion
                               "Supported repositories: #{SUPPORTED_REPOS.map { |t| "`#{t}`" }.to_sentence}." \
```

looks pretty good

<img width="632" alt="Screenshot 2022-08-03 at 18 25 27" src="https://user-images.githubusercontent.com/355033/182671137-9ffd8183-5b0a-4035-9f3f-2cf0a84bf923.png">